microsoft/testsuites/core/azure_image_standard.py

```diff
@@ -1583,3 +1613,22 @@ def _verify_version_by_pattern_value(
                 raise LisaException(message + action_message)
             elif not extended_support_versions:
                 raise LisaException(message + action_message)
+
+    def _get_not_enabled_modules(self, node: Node) -> List[str]:
+        """
+        Returns the list of essential kernel modules that are neither integrated
+        into the kernel nor compiled as loadable modules.
+        """
+        # These modules are essential for Hyper-V / Azure platform.
+        essential_modules_configuration = {
+            "wdt": "CONFIG_WATCHDOG",
+            "cifs": "CONFIG_CIFS",
+        }
```

Comment:
I would still recommend having this as a class variable like "linux_essential_modules". Someone looking at the test case will immediately understand the modules being verified.

If it's defined within a function, they will need to navigate through the functions